content/blog/2018-04-22-essurvey.md

```diff
@@ -0,0 +1,337 @@
+    ##    <chr> <chr>  <dbl>
+    ##  1 ESS1  ES        0.
+    ##  2 ESS1  ES        0.
+    ##  3 ESS1  ES        5.
+    ##  4 ESS1  ES        5.
+    ##  5 ESS1  ES        4.
+    ##  6 ESS1  ES        7.
+    ##  7 ESS1  ES        2.
+    ##  8 ESS1  ES        2.
+    ##  9 ESS1  ES        3.
+    ## 10 ESS1  ES        3.
+    ## # ... with 26,514 more rows
+
+Now that we've got that down, let's visualize the change over time by
+calculating the percentage of respondents in each category for every
+year/country combination and visualize the results.
+
+    semi_cleaned %>% 
+      mutate(stfgov = case_when(stfgov <= 3 ~ "Low",
+                                between(stfgov, 4, 6) ~ "Mid",
+                                stfgov >= 7 ~ "High"),
+             stfgov = factor(stfgov, levels = c("Low", "Mid", "High"))) %>%
+      count(name, cntry, stfgov) %>% 
+      group_by(name, cntry) %>% 
+      mutate(perc = n / sum(n)) %>% 
+      ggplot(aes(name, perc, group = stfgov, colour = stfgov)) +
+      geom_line() +
+      theme_bw() +
+      labs(x = "ESS rounds", y = "Satisfaction with Government (%)") +
+      scale_colour_discrete(name = NULL) +
+      facet_wrap(~ cntry)
+
+![](/img/blog-images/2018-04-22-essurvey/spain_fr_stf-1.png)
+
+Spain and France follow very similar patterns although the changes are
+much steeper in Spain! A more elaborate analysis would perhaps be
+interested in finding out if this trend line is similar in other
+countries.
+
+Downloading data for Stata, SPSS and SAS users
+----------------------------------------------
+
+To finish off, all `import_*` functions have an equivalent `download_*`
+function that allows the user to save the datasets in a specified folder
+in `'stata'`, `'spss'` or `'sas'` formats.
+
+For example, to save round two from Turkey in a folder called
+`./my_folder`, we use:
+
+    download_country("Turkey", 2,
+                     output_dir = "./myfolder/")
+
+By default it saves the data as `'stata'` files. Alternatively you can
+use `'spss'` or `'sas'`.
+
+    download_country("Turkey", 2,
+                     output_dir = "./myfolder/",
+                     format = 'sas')
+
+This will save the data to `./myfolder/ESS_Turkey` and inside that
+folder there will be the `ESS2` folder that contains the data. The round
+equivalent is `download_rounds`
+
+Analyzing ESS data
+------------------
+
+Be aware that for analyzing data from the ESS survey you should take
+into consideration the sampling and weights of the survey. The
+[survey](http://r-survey.r-forge.r-project.org/survey/) package provides
+very good support for this. A useful example comes from the work of
+Anthony Damico and Daniel Oberski
+[here](http://asdfree.com/european-social-survey-ess.html). This example
+calculated percentages manually and **appropriate statistical
+inference** should consider the above.
+
+Acknowledgments
+---------------
+
+The package was improved greatly thanks to the reviews of [Thomas

```

Comment:
link to package repo here? 
If you're interested, this is an opportunity to tell people how they could contribute.